cmd/create.go

```diff
@@ -67,6 +72,13 @@ func create() {
 		return
 	}
 
+	// Check if recertContainerImage was provided by the user
+	if recertContainerImage == "" {
+		fmt.Printf(" *** Please provide a valid container image for the recert tool *** \n")
+		log.Info("Skipping OCI image creation.")
+		return
```

Comment:
This seems like a good place to use `log.Fatal`

```suggestion
		log.Fatal"*** Please provide a valid container image for the recert tool *** \n")
```

But maybe in a separate PR, because this pattern is already commonplace in the codebase